README.md

```diff
@@ -7,17 +7,47 @@ localhost.
 The goal of ccm and ccmlib is to make it easy to create, manage and destroy a
 small Cassandra cluster on a local box. It is meant for testing a Cassandra cluster.
 
+New to Python development?
+--------------------------
+Python has moved on since CCM started development. `pip` is the new `easy_install`,
+Python 3 is the new 2.7, and virtualenv is strongly recommended for managing multiple Python versions
```

Comment:
Suggest: Python 3.7 or later is required and using virtualenv is strongly recommended for managing multiple Python versions